upb/reflection.h

```diff
@@ -134,6 +134,19 @@ void upb_array_set(upb_array *arr, size_t i, upb_msgval val);
 /* Appends an element to the array.  Returns false on allocation failure. */
 bool upb_array_append(upb_array *array, upb_msgval val, upb_arena *arena);
 
+
+/* Inserts one or more empty elements into the array.  Existing elements are
+ * shifted right.  The new elements have undefined state and must be set with
+ * upb_array_set().
```

Comment:
nit: backticks on code in comments please.

Reply:
Done.